src/views/Results.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <section>
+    <button @click="selectView">List View</button>
+    <button @click="selectView">Map View</button>
+    <results-list v-if="listView"></results-list>
+    <results-map v-else></results-map>
+  </section>
+</template>
+
+<script>
+import ResultsList from '../components/ResultsList.vue'
+import ResultsMap from '../components/ResultsMap.vue'
+export default {
+  data() {
+    return {
+      listView: true,
+    };
+  },
+  components: {
```

Comment:
Adding this components option was critical to getting our child components to display. Thanks for this fix, Stacy!

Reply:
Thanks for this heads up - we ran into something similar when figuring out when this is or isn't necessary to render a component.  It seems as though if a component is set up in the routes array as part of an object, it'll render automatically.  If not, we need to export the components object the way you have it here.  That's my current understanding!

Reply:
This definitely sounds right. This will keep them on the `/results` route while allowing them to swap between the `results-list` and `results-map` components. To my understanding of Vue, a parent component only becomes aware of any child components it has to keep track of when this components option is declared and defined.